README.md

```diff
@@ -1,10 +1,10 @@
 ![Django Backend](https://github.com/Zorroa/scrounger/workflows/Django%20Backend/badge.svg) ![React Frontend](https://github.com/Zorroa/scrounger/workflows/React%20Frontend/badge.svg)
 # What is Scrounger?
 
-Scrounger is an open source content search application built by [Zorroa](https://zorroa.com), the makers of a GUI-driven machine learning integration platform. The app is designed to let its users experience what it’s like to load their media assets and run an ML-powered  search within a custom app, in under an hour.
+Scrounger is an open source content search application built by [Boon AI](https://zorroa.com), the makers of a GUI-driven machine learning integration platform. The app is designed to let its users experience what it’s like to load their media assets and run an ML-powered  search within a custom app, in under an hour.
```

Comment:
What about a slight reword? It's inaccurate to say it's built by Boon AI bc of the whole dba discussion, no?

```suggestion
Scrounger is an open source content search application built by the makers of [Boon AI](https://zorroa.com), a GUI-driven machine learning integration platform. Scrounger is designed to let its users experience what it’s like to load their media assets and run an ML-powered  search within a custom app, in under an hour.
```

Reply:
I like this, but I feel like that's the part where @danny-zorroa wants to involve marketing/copywriting, so let's see what he says.